routes/api.php

```diff
@@ -13,6 +11,10 @@
 |
 */
 
-// Route::middleware('auth:api')->get('/user', function (Request $request) {
```

Comment:
I am thinking the intent is to create a minimal prep for supporting API's in a similar folder structure to the rest of boilerplate without impacting to much.   The line was commented in boilerplate.